armi/bookkeeping/report/html.py

```diff
@@ -264,8 +264,6 @@ def writeStandardReportTemplate(f, report):
                 with Div(f, attrs={"class": "page-header"}):
                     with H1(f):
                         f.write(report.title)
-                    with P(f):
-                        f.write(cs["comment"])
```

Comment:
Don't know if you care or noticed, but just pointing out here that you removed functionality which I think _did_ work before.

Maybe you meant to add that as an argument as well?

Reply:
Well, I can add it back in.  

But I couldn't find that many examples of people using this code, and the examples I _did_ find had empty "comment" settings.

If you can find me (offline) a counter example of someone putting a comment into a report, I can put it back in.  I just thought this looked like a mostly unused system, and I was trying to make it easier to support long term. 

I'm not married to this one change, if you have reason for me to put it back in.

Reply:
I don't really know if people use the `comment` case setting anyways haha. 

I don't particularly care if it is removed, just seemed unnecessary. If you're willing to deal with whomever this might impact (if anyone), then go for it.